upi/openstack/inventory.yaml

```diff
@@ -6,7 +6,7 @@ all:
 
       # User-provided values
       os_subnet_range: '10.0.0.0/16'
-      os_cluster_name: 'openshift'
+      os_cluster_name: "{{ lookup('env', 'INFRA_ID') }}"
```

Comment:
The idea here is that the user provides values in this paragraph. What is the use case for variables in env?

Reply:
I just figured it would be easy, since we already tell the user to source this value. But I'll tell you what, I can change it to `{{ lookup('env', 'INFRA_ID') | default('openshift') }}`

Reply:
The difference is, that the installer will always define this value as <cluster-name>-<random hash>, so this at least makes sure that then name on your configs matches the name on your infra. It also depends on the workflow you want here. If we are going `install-config` --> inventory, as the docs currently lay it out, it makes sense this way. However, if we go 
inventory --> install-config    then it makes sense to have a user defined cluster name here, and then read in the INFRA_ID elsewhere. Ultimately though, somewhere in the inventory, we will have to read this value back from the installer metadata.

Reply:
FWIW I'd prefer if the inventory/playbooks read it from install-config. Less error-prone that way.

Also I believe, there's a difference between *cluster name* and *infra ID*. Cluster name is the name you specify in install-config.yaml / installer prompt (e.g. `lolcloud`) but infra-id is cluster name *plus* the random suffix (e.g. `lolcloud-23n9`).

So I think the simplest workflow we can do would be to have `os_cluster_name` as a user-defined variable in the inventory and read `os_infra_id` from install-config (or rather, `metadata.json`).